src/ray/stats/metric_defs.cc

```diff
@@ -169,6 +169,14 @@ DEFINE_stats(scheduler_failed_worker_startup_total,
              (),
              ray::stats::GAUGE);
 
+/// Raylet Resource Manager
+DEFINE_stats(resources,
+             // TODO(sang): Support placement_group_reserved_available | used
+             "Logical Ray resources broken per state {AVAILABLE, USED}",
+             ("Name", "State"),
+             (),
+             ray::stats::GAUGE);
```

Comment:
Feel free to punt this to later, but it would be nice to add a IN_PLACEMENT_GROUP breakdown as well.

Reply:
that will be done in a follow up PR!